doc/rules/control_structure/trailing_comma_in_multiline.rst

```diff
@@ -2,8 +2,8 @@
 Rule ``trailing_comma_in_multiline``
 ====================================
 
-Multi-line arrays, arguments list, parameters list and ``match`` expressions
-must have a trailing comma.
+Multi-line arrays, arguments list, parameters list,  ``match`` and array
```

Comment:
What about sorting the elements in `setAllowedValues` call in `TrailingCommaInMultilineFixer`, so these here are in the same order as in line 26?

Reply:
Thanks, I think I've done what you are suggesting, let me know if I misunderstood and I'll adjust accordingly

Reply:
I was thinking to sort them alphabetically by value, so arguments option goes before both array options and params option is last

Reply:
thanks updated :+1: